clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -173,24 +194,22 @@ public void testHeartbeatResponseOnErrorHandling(final Errors error, final boole
         heartbeatRequestState = new HeartbeatRequestManager.HeartbeatRequestState(
             mockLogContext,
             mockTime,
-            heartbeatInterval,
+            heartbeatIntervalMs,
             retryBackoffMs,
             retryBackoffMaxMs,
             0);
-        when(mockMembershipManager.state()).thenReturn(STABLE);
         heartbeatRequestManager = createManager();
 
-        // Sending first heartbeat to set the state to STABLE
+        // Sending first heartbeat w/o assignment to set the state to STABLE
         ConsumerGroupHeartbeatResponse rs1 = new ConsumerGroupHeartbeatResponse(new ConsumerGroupHeartbeatResponseData()
-            .setHeartbeatIntervalMs(heartbeatInterval)
+            .setHeartbeatIntervalMs(heartbeatIntervalMs)
             .setMemberId(memberId)
-            .setMemberEpoch(memberEpoch)
-            .setAssignment(memberAssignment));
```

Comment:
seems we're not using `memberAssignment` in the test anymore? let's remove if unused